packages/web-twig/tests/__snapshots__/ComponentsSnapshotTest__test with data set paginationDefault.twig__1.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <title>
+    </title>
+  </head>
+  <body>
+    <nav role="navigation">
+      <ul class="Pagination">
+        <li class="Pagination__item">
+          <a class="Pagination__link Pagination__link--current" href="&amp;#x23;" aria-current="page"><span class="accessibility-hidden">Current Page, Page 1</span> <span aria-hidden="true">1</span></a>
+        </li>
+
+        <li class="Pagination__item">
+          <a class="Pagination__link" href="&amp;#x23;"><span class="accessibility-hidden">Goto Page 2</span>
```

Comment:
Is it really "Goto", not "Go to"…?

Reply:
According to many dictionaries, it is a "Go to". I fixed that.

Reply:
Please see also https://github.com/lmc-eu/spirit-design-system/pull/895 where I fixed this for the web package.